HoI4Mod/Rome_Strikes_Back/events/TESB_AOT.txt

```diff
@@ -1963,4 +1963,163 @@ news_event = {
 	option = {
 		name = OK
 	}
+}
+
+
+# outside support
+country_event = {
+	id = tesb_AOT.2500
+	title = tesb_AOT.2500.t
+	desc = tesb_AOT.2500.desc
+	picture = GFX_report_event_japanese_parade
+	is_triggered_only = yes
+
+	option = {
+		name = tesb_AOT.2500.a
+		set_country_flag = support_AOT
+		hidden_effect = {
+			AOT = { country_event = { id = tesb_AOT.2501 random_hours = 48 } }
+			add_ideas = AOT_volunteers
+		}
+		ai_chance = {
+			base = 0
+			modifier = { add = 70 tag = MNG }
+			modifier = { add = 60 tag = KOR }
+			modifier = { add = 50 tag = UTA }
+			modifier = { add = 50 tag = MEX }
+			modifier = { add = 50 tag = ENG }
+			modifier = { add = 50 tag = CAN }
+			modifier = { add = 50 tag = BIR }
+			modifier = { add = 30 tag = JAP }
+			modifier = { add = 30 tag = AST }
+			modifier = { add = 30 tag = RME }
+			modifier = { add = 30 tag = IND }
+			modifier = {
+				factor = 0.5
+				has_war = yes
+			}
+		}
+	}
+	option = {
+		name = tesb_AOT.2500.b
+		set_country_flag = support_D01
+		hidden_effect = {
+			D01 = { country_event = { id = tesb_AOT.2501 random_hours = 48 } }
```

Comment:
As before, D01 won't be guaranteed to be AOT's revolt, better to use the proper `random_country_with_original_tag` format